WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -672,6 +673,7 @@ private extension ProductFormViewController {
                     return
                 }
 
+                self?.trackVariationRemoveButtonTapped()
                 self?.showVariationDeletionProgress()
                 self?.deleteProduct()
```

Comment:
Something I noticed while deleting variations:

The tracks event for `trackVariationRemoveButtonTapped()` above works, but then when we call `deleteProduct()` here it fires `product_detail_product_deleted`. That is an existing tracks event, and I think we try not to change tracks events that are in use (right?), but is that sufficiently clear when we're deleting a variation versus a product?

Reply:
> That is an existing tracks event, and I think we try not to change tracks events that are in use (right?)

That's right!

> but is that sufficiently clear when we're deleting a variation versus a product?

When they added that metric I don't think they care about what kind of product was being deleted. So I think we are safe here, as our event is enough for us!